website/source/docs/runtime/interpolation.html.md.erb

```diff
@@ -201,4 +201,39 @@ a particular node and as such can not be used in constraints.
 Environment variables should be enclosed in brackets `${...}` for
 interpolation.
 
+### Dots in Variables <a id="dots_in_vars"></a>
+
+Starting in Nomad 0.9, task configuration interpolation requires variables to
+be valid identifiers. While this does not affect default variables or common
+custom variables, it is possible to define a variable that is not a valid
+identifier:
+
+```hcl
+env {
+  "valid.name"     = "ok"
+  "invalid...name" = "not a valid identifier"
+}
+```
+
+The environment variable `invalid...name` cannot be interpolated using the
+standard `${"invalid...name"}` syntax. The dots wil be interpreted as object
```

Comment:
I think this example was meant to be `"${invalid...name}"` (quotes on the outside).

Reply:
Fixed! Thanks!